promql/parser/lex.go

```diff
@@ -313,6 +313,11 @@ func (l *Lexer) accept(valid string) bool {
 	return false
 }
 
+// is peeks and returns true if the next rune is in the valid set.
```

Comment:
From the doc comment, I found it a bit hard to understand what this helper method is doing. I had to read the code. Given that the code is short and the method is unexported, I would be fine without a doc comment. But if we keep it, it should probably be clearer, e.g. "is is peeks and returns true if if the next rune is contained in the provided string".

Reply:
Patched, PTAL.